homeassistant/components/dht/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "dht",
   "name": "DHT Sensor",
   "documentation": "https://www.home-assistant.io/integrations/dht",
-  "requirements": ["Adafruit-DHT==1.4.0"],
+  "requirements": ["adafruit-circuitpython-dht==3.6.0"],
   "codeowners": [],
```

Comment:
You can remove lines with empty values. There was a manifest cleanup recently: https://github.com/home-assistant/core/pull/49745 or add yourself as a code owner.

Reply:
This can be a follow up.